src/main/java/com/minecolonies/core/entity/pathfinding/PathfindingUtils.java

```diff
@@ -331,7 +329,7 @@ public static boolean isLava(@NotNull final BlockGetter world, final BlockPos po
      */
     public static boolean isLadder(final BlockState blockState, @Nullable final PathingOptions options)
     {
-        return blockState.is(BlockTags.CLIMBABLE) && ((options != null && options.canClimbNonLadders()) || blockState.getBlock() instanceof LadderBlock);
+        return blockState.is(BlockTags.CLIMBABLE) && ((options != null && options.canClimbNonLadders()) || blockState.is(ModTags.freeClimbBlocks));
```

Comment:
What about modded blocks that implement a ladder, those will seize to work now.
Additionally, should it respect that `options.canClimbNonLadders`?

Also the method probably should be renamed if this is to be considered.

Reply:
That's why there's a tag, they had to be in the `CLIMBABLE` tag anyway, now they need to be in the other tag too -- or they're still climbable, but need the research.

But yes, I could rename the method.  I wanted it to be a smaller change for now 😁

Reply:
Because using a tag is better than hard-coding things, because modpacks can extend tags and they can't extend not-tags.

Reply:
instanceof ladder is intentional here for automatic mod compat

Reply:
You can technically just have both can't you? `blockState.getBlock instanceof LadderBlock || blockState.is(ModTags.freeClimbBlocks)`

Reply:
Are there mods that derive from LadderBlock?

But they'd still have to be added to `CLIMBABLE` explicitly anyway.  Though yeah, it could check both.

Reply:
ye there is a bunch of mods that do that for sure^^